src/CashFlows.py

```diff
@@ -258,6 +266,14 @@ def getTax(self):
     """
     return self._tax
 
+  def getOutput(self):
+    """
+    Get output Type
```

Comment:
these docstring lines should be tabbed in 2 spaces, 1 indent in from the triple-quotes